server/aap/aaprss/views.py

```diff
@@ -0,0 +1,15 @@
+from superdesk.core.web import EndpointGroup
+from superdesk.core.types import Request, BaseModel, Response
+
+from aap.formatters.aaprss import AAPRSSFormatter
+
+aap_rss_endpoints = EndpointGroup("aaprss", __name__)
+
+
+class RSSArgs(BaseModel):
+    days: str | None = "2"
```

Comment:
Is there a reason to use `str` here?
You can use `int` and Pydantic will validate and serialise the provided value for us